proto/source_metadata.proto

```diff
@@ -331,6 +331,18 @@ message Elasticsearch {
   string timestamp = 3;
 }
 
+message Sentry {
+  string id = 1;
```

Comment:
I would prefer this to be named event_id so we know what it represents.

Reply:
Oof, dunno why I did that, will change 👍